python_agent/README.md

```
@@ -83,12 +86,14 @@ All responses from the Java plugin are JSON strings.
 
 ## Python Agent Setup
 
-1.  **Create a virtual environment (recommended):**
+1. **Create a virtual environment (recommended):**
+
     ```bash
     python -m venv .venv
     source .venv/bin/activate  # On Windows: .venv\Scripts\activate
     ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)